lib/cog/adapters/hipchat/api.ex

```diff
@@ -14,7 +14,7 @@ defmodule Cog.Adapters.HipChat.API do
   end
 
   def lookup_room("@" <> handle) do
-    {:ok, user} = lookup_user(name: "@" <> handle)
+    {:ok, user} = lookup_user(handle: handle)
```

Comment:
I've been following the conventions from Elixir's source. One-liner and smaller functions logically similar functions are grouped together, but most of the time there are newlines between functions. I think the idea is to remove newlines to say "these functions are logically similar and could be replaced by a simple case statement", while adding a newline means "these functions use pattern matching but have totally different implementations and could potentially be given unique names".